docs/source/nodeoperations/restore.md

```diff
@@ -2,88 +2,216 @@
 
 This procedure will describe how to restore from backup taken using [Scylla Manager](../manager.md) to a fresh **empty** cluster of any size.
 
-First identify to which snapshot you want to restore. To get list of available snapshot execute following command on Scylla Manager Pod.
-```bash
-sctool backup list -c <CLUSTER_ID> --all-clusters -L <BACKUP_LOCATION>
-```
+:::{caution}
+Due to a [bug](https://github.com/scylladb/scylla-manager/issues/3679) in Scylla Manager not supporting ScyllaClusters having both non-TLS and TLS CQL ports open, you have to disable the TLS certificate management
+in Scylla Operator.
+This step will no longer be required, when the bug is fixed.
 
-Where:
-* `CLUSTER_ID` - is a name of a cluster or ID under which ScyllaCluster was registered. You can find it in ScyllaCluster Status.
-* `BACKUP_LOCATION` - is a location where backup is stored. For example, for bucket called `backups` stored in AWS S3, location is `s3:backups`.
+To disable TLS certificate management in Scylla Operator add `--feature-gates="AutomaticTLSCertificates=false"` flag to Scylla Operator deployment.
 
-```bash
-sctool backup list -c simple-cluster --all-clusters -L s3:backups
-Snapshots:
-  - sm_20201227144037UTC (409MiB)
-  - sm_20201228145917UTC (434MiB)
-Keyspaces:
-  - users (9 tables)
-  - system_auth (2 tables)
-  - system_distributed (3 tables)
-  - system_schema (13 tables)
-  - system_traces (5 tables)
+```console
+kubectl -n scylla-operator patch deployment/scylla-operator --type=json -p='[{"op": "add", "path": "/spec/template/spec/containers/0/args/-", "value": "--feature-gates="AutomaticTLSCertificates=false"}]'
```

Comment:
This failed for me - the doublequotes are invalid.
```suggestion
kubectl -n scylla-operator patch deployment/scylla-operator --type=json -p='[{"op": "add", "path": "/spec/template/spec/containers/0/args/-", "value": "--feature-gates=AutomaticTLSCertificates=false"}]'
```

Reply:
Thanks, fixed